custom-connectors/Actsoft/apiDefinition.swagger.json

```diff
@@ -0,0 +1,14408 @@
+                  "Message": "Service type definition not found."
+                },
+                {
+                  "Code": 105,
+                  "Message": "Field caption not found."
+                },
+                {
+                  "Code": 106,
+                  "Message": "Duplicate field captions found."
+                },
+                {
+                  "Code": 107,
+                  "Message": "Company dispatch module does not exist."
+                },
+                {
+                  "Code": 108,
+                  "Message": "User with 'Employee Number' not found."
+                },
+                {
+                  "Code": 109,
+                  "Message": "Order with ExternalId already exists."
+                },
+                {
+                  "Code": 110,
+                  "Message": "ExternalClientId is required."
+                },
+                {
+                  "Code": 111,
+                  "Message": "ExternalOrderId is required."
+                },
+                {
+                  "Code": 112,
+                  "Message": "OrderDate is required."
+                },
+                {
+                  "Code": 113,
+                  "Message": "ServiceTypeName is required."
+                },
+                {
+                  "Code": 114,
+                  "Message": "User with 'Employee Number' already deleted or deactivated."
+                },
+                {
+                  "Code": 115,
+                  "Message": "Required form field not found."
+                },
+                {
+                  "Code": 116,
+                  "Message": "Contact Phone length is not valid."
+                },
+                {
+                  "Code": 117,
+                  "Message": "Contact Name length is not valid."
+                },
+                {
+                  "Code": 118,
+                  "Message": "Contact Email format is not valid."
+                },
+                {
+                  "Code": 119,
+                  "Message": "Latitude\\Longitude values are not valid."
+                },
+                {
+                  "Code": 120,
+                  "Message": "Country not supported."
+                },
+                {
+                  "Code": 121,
+                  "Message": "Region not supported."
+                },
+                {
+                  "Code": 122,
+                  "Message": "DueDateFrom is required."
+                },
+                {
+                  "Code": 123,
+                  "Message": "DueDateTo is required."
+                },
+                {
+                  "Code": 124,
+                  "Message": "ExternalId length is not valid."
+                },
+                {
+                  "Code": 125,
+                  "Message": "Contact Email length is not valid."
+                },
+                {
+                  "Code
```

Comment:
@rdameron Your swagger has unicodes which we do not support at the moment, Can you please replace them with English language characters.

Reply:
I could not find unicode in the file.